fusion-plugin-react-router/src/types.js

```diff
@@ -8,6 +8,24 @@
 
 import * as React from 'react';
 
+/* Custom types */
+export type RouterPropsType = {
+  context?: any,
+  onRoute?: Function,
+  history: RouterHistoryType,
+  Provider?: BaseRouterType,
+  basename?: string,
+  children?: React.Node,
+};
+
+export type RouterType = React.ComponentType<RouterPropsType>;
+
+/* Types below adapted from flow-typed's libdef for react-router-dom
```

Comment:
Updated all of the below types to mirror those from the libdef.  Ideally, these remain in sync.